py_examples/helix_case/helixExample.py

```diff
@@ -51,7 +54,14 @@
 gravity_force = py_dismech.GravityForce(soft_robots, np.array([0.0, 0.0, -9.8]))
 add_force(gravity_force)
 
+
+
 # Initialize and run the simulation
 sim_manager.initialize(sys.argv)
+time = 0.0
 while not sim_manager.simulation_completed():
-    sim_manager.step_simulation()
+    # sim_manager.step_simulation()
+
+    input_dict = {"position" : np.array([[0, 0, time * 0.01, 0.0, 0.0], [0, 1, time * 0.01, 0.0, 0.0]])}
+    sim_manager.step_simulation(input_dict)
+    time += sim_params.dt
```

Comment:
Running
```
pip install pre-commit
pre-commit install
pre-commit run --all-files
```
will fix indentations eof issues like this.

Reply:
Need more discussion later

Reply:
Sure, we can discuss offline.